linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2/oval/shared.xml

```diff
@@ -0,0 +1,40 @@
+<def-group>
+  <definition class="compliance" id="accounts_passwords_pam_tally2" version="4">
+    <metadata>
+      <title>Lock out account after failed login attempts</title>
+      <affected family="unix">
+        <platform>multi_platform_sle</platform>
+      </affected>
+      <description>The number of allowed failed logins should be set correctly.</description>
+    </metadata>
```

Comment:
Please use the macro `{{{ macro oval_metadata(description, title="", affected_platforms=None) }}}` here.

You need to define only the `description`. Rule title and affected platforms will be automatically be picked up by the macro processing.
```suggestion
    {{{ macro oval_metadata("The number of allowed failed logins should be set correctly.") }}}
```

Reply:
Can you please make this change?

Reply:
will do, sorry I missed it before.

Reply:
no problem